### PR TITLE
fix(charts): correctly template off-cluster redis credentials

### LINKED ISSUE #127: v2.9.0 logger still does not work with off-cluster redis
`$ kubectl -n deis logs deis-logger-3305119006-ixc1p
2016/12/06 11:44:46 config error: envconfig.Process: assigning LOGGER_DEIS_LOGGER_REDIS_SERVICE_PORT to DEIS_LOGGER_REDIS_SERVICE_PORT: converting 'redis port' to type int:`

### PR DESCRIPTION
**Pull request by bacongobbler**
closes deis/logger#127

### CODE REVIEW

File: charts/redis/templates/logger-redis-secret-creds.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
     "helm.sh/hook": pre-install
 data: {{ if eq .Values.global.logger_redis_location "on-cluster"}}
   password: {{ randAlphaNum 32 | b64enc }}{{ else if eq .Values.global.logger_redis_location "off-cluster"}}
-  db: {{ .Values.db | b64enc }}
-  password: {{ .Values.password | b64enc }}
-  host: {{ .Values.host | b64enc }}
-  port: {{ .Values.port | b64enc }}{{ end }}
+  db: {{ .Values.logger.redis.db | b64enc }}
```
Comment (kmala) [inline charts/redis/templates/logger-redis-secret-creds.yaml]: this doesn't work as the chart is under redis directory....i think the better fix would be removing the logger from https://github.com/deis/workflow/blob/master/charts/workflow/values.yaml#L125

Comment (rimusz) [reply charts/redis/templates/logger-redis-secret-creds.yaml]: agree with @kmala 
I did the same for `fluentd` syslog endpoint https://github.com/deis/workflow/pull/631/commits/67b658f692c65e274b486e7f3e91705bd1204921#diff-d3168eb1e6c9201c7558cc356cd625daR135

Comment (bacongobbler) [reply charts/redis/templates/logger-redis-secret-creds.yaml]: That sounds good to me, considering that this didn't work in the first place :)

**Comment by bacongobbler** [comment]
closing in favour of https://github.com/deis/workflow/pull/643.

**Pull request closed without merging**